### PR TITLE
Included contributing section into README:

### PR DESCRIPTION
**Pull request by fsenf**
Based on the discussions in PR #10 & issue #8, I added a small description to our README containing:
(1) requirements for PRs
(2) example data upload

### CODE REVIEW

**Comment by JuliaKukulies** [comment]
Great! Maybe the PR requirements could also be in form of a [pull_request_template.md](https://github.com/climate-processes/tobac/pull/120), to be consistent with the other repo and that these checks will automatically appear when submitting a PR?

**Comment by JuliaKukulies** [comment]
Great! Maybe the PR requirements could also be in form of a [pull_request_template.md](https://github.com/climate-processes/tobac/pull/120), to be consistent with the other repo and that these checks will automatically appear when submitting a PR?

**Comment by fsenf** [comment]
@JuliaKukulies: Nice idea! I added a PR template based on your link!

**Comment by fsenf** [comment]
@JuliaKukulies : Would you like to proceed the review?

File: README.md [hunk 1]
```diff
@@ -13,3 +13,38 @@ A step-by-step description to run the tutorials with jupyter is located [here](d
...
+* **with your example notebook included**
+  * please extensively use markdown elements to structure and explain your notebook
+  * check already existing notebooks to get an idea about our tutorial style
+  * please upload your example data to [zenodo]( https://zenodo.org/ ) (see next paragraph)
+  * please check if your notebook is running in the recommended standard conda enviroment [see here]((docs/RunningTutorials.md) and update the [list of required python packages](./requirements.txt) if needed
+
+* **with your suggestions to change the documentation elements**, e.g. `index.rst`
+  * please care that sphinx-based rendering is still working correctly
+
+* **or with combination of both**
```
Comment (JuliaKukulies) [inline README.md]: I think this line would slightly confuse me, should there be separate pull requests or is it OK to have the addition of the notebook and changes to `index.rst` in one?

File: README.md [hunk 2]
```diff
@@ -13,3 +13,38 @@ A step-by-step description to run the tutorials with jupyter is located [here](d
...
+Please provide a separate pull request:
+
+* **with your example notebook included**
+  * please extensively use markdown elements to structure and explain your notebook
+  * check already existing notebooks to get an idea about our tutorial style
+  * please upload your example data to [zenodo]( https://zenodo.org/ ) (see next paragraph)
+  * please check if your notebook is running in the recommended standard conda enviroment [see here]((docs/RunningTutorials.md) and update the [list of required python packages](./requirements.txt) if needed
+
+* **with your suggestions to change the documentation elements**, e.g. `index.rst`
+  * please care that sphinx-based rendering is still working correctly
```
Comment (JuliaKukulies) [inline README.md]: Maybe provide a link on how to check this or in general how jupyter notebook works with Sphinx (e.g. https://docs.readthedocs.io/en/stable/guides/jupyter.html) ?

**Review by JuliaKukulies — APPROVED** [review]
Hey @fsenf, two small comments from me (see below). Otherwise it looks good, also with the PR template. Thanks for adding this one! I hope it is not an overkill of instructions now, but I think it is useful to have this PR check list to facilitate the contribution and review.

**Comment by fsenf** [comment]
@JuliaKukulies : Thanks for your suggestions! I removed the bullet that confused you & I added a small docu on how to run sphinx-based rendering locally. Please have a look ...

**Comment by fsenf** [comment]
@JuliaKukulies : PR ready to merge? What do you think?

**Comment by JuliaKukulies** [comment]
Sorry for delaying this @fsenf, but I am happy with it now and would say go ahead and merge!

**Comment by fsenf** [comment]
@JuliaKukulies : Great, and be convinced that I am always slower than you ;-)
